main.js

```diff
@@ -0,0 +1,87 @@
+// create variable to get elements from HTML file
+const hole = document.getElementById("hole");
+const bird = document.getElementById("bird");
+let isBirdJumping = 0;  // Flag to determine if the bird is isBirdJumping
+let score = 0; // Initialize the score
+
+// Changing the position of hole after every animations
+hole.addEventListener('animationiteration' , () => {
+    // Process: Generate a random position for the hole's top between 200px and 450px

```

Comment:
CS1: những thông tin như min/max hole top position, starting position of bird, game over threshold là constant do đó em nên define constant variables ở trên cùng sau này mình muốn sửa logic adjust các thông tin này thì chỉ cần sửa ở constant sẽ dễ hơn